doc/releases/release-notes-4.1.rst

```diff
@@ -547,6 +548,7 @@ New Drivers
    * :dtcompatible:`nxp,ii2c`
    * :dtcompatible:`ti,omap-i2c`
    * :dtcompatible:`ti,tca9544a`
+   * :dtcompatible:`nordic,nrf-twis`
```

Comment:
nit, since you will have to update anyway, please sort alphabetically (i.e put it first)

Reply:
done

Reply:
done